jerry-core/ecma/base/ecma-helpers.h

```diff
@@ -137,6 +137,18 @@ typedef enum
  */
 #define ECMA_BOOL_TO_BITFIELD(x) ((x) ? 1 : 0)
 
+#ifndef CONFIG_DISABLE_ES2015_SYMBOL_BUILTIN
+/**
+ * JERRY_ASSERT compatible macro for checking whether the given ecma-value is symbol
+ */
+#define ECMA_ASSERT_VALUE_IS_SYMBOl(value) (ecma_is_value_symbol ((value)))
```

Comment:
Typo: `ECMA_ASSERT_VALUE_IS_SYMBOl` -> `ECMA_ASSERT_VALUE_IS_SYMBOL`

Reply:
Good catch, I've added to the TODO list. I'm going to apply your suggestions after the PR got an internal review.